web/packages/teleterm/src/ui/AppUpdater/DetailsView.tsx

```diff
@@ -0,0 +1,265 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useId } from 'react';
+
+import {
+  Alert,
+  ButtonPrimary,
+  ButtonSecondary,
+  Flex,
+  Indicator,
+  Link,
+  P2,
+  P3,
+  Stack,
+  Text,
+} from 'design';
+import { Checks, Info } from 'design/Icon';
+
+import { Platform } from 'teleterm/mainProcess/types';
+import { AppUpdateEvent, UpdateInfo } from 'teleterm/services/appUpdater';
+import { RootClusterUri } from 'teleterm/ui/uri';
+
+import { AutoUpdatesManagement } from './AutoUpdatesManagement';
+import {
+  ClusterGetter,
+  formatMB,
+  iconMac,
+  iconWinLinux,
+  isTeleportDownloadHost,
+} from './common';
+
+/**
+ * Detailed updates view.
+ * Shows the auto updates status and
+ */
+export function DetailsView({
```

Comment:
Could you add JSDocs for those main components explaining how they can be accessed? Similar to how I done it in the stories.